core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -349,26 +349,25 @@ public Configuration defineConfiguration(String name, String template, Configura
          if (c == null) {
             throw log.undeclaredConfiguration(template, name);
          } else {
-            return defineConfiguration(name, false, configurationOverride, c);
+            return actualDefineConfiguration(name, configurationOverride, c);
```

Comment:
Hrmm maybe.  We can log a different JIRA for it me thinks, all I did was remove the boolean.